web/src/engine/websites/CoroCoro.ts

```diff
@@ -1,21 +1,59 @@
 import { Tags } from '../Tags';
 import icon from './CoroCoro.webp';
-import { DecoratableMangaScraper } from '../providers/MangaPlugin';
-import * as CoreView from './decorators/CoreView';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
 import * as Common from './decorators/Common';
+import { FetchCSS } from '../platform/FetchProvider';
+import type { Priority } from '../taskpool/DeferredTask';
+import { Exception } from '../Error';
+import { WebsiteResourceKey as R } from '../../i18n/ILocale';
 
-function MangaExtractor(element: HTMLAnchorElement) {
-    return {
-        id: element.pathname,
-        title: element.querySelector('div.p-article-image img').getAttribute('alt').trim()
-    };
+type JSONMangas = {
+    weekdays: {
```

Comment:
May use keyed type such as Record
```typescript
type WeekDays = 'mon' | 'tue' | 'wed';
type Mangas = Record<WeekDays, Manga[]>;
```

Reply:
On second thought, do the weekdays really matter?
May just use `type Mangas = Record<string, Manga[]>;`